student/10th/kimdohwan/westagram/config/urls.py

```diff
@@ -0,0 +1,23 @@
+"""conifg URL Configuration
+
+The `urlpatterns` list routes URLs to views. For more information please see:
+    https://docs.djangoproject.com/en/3.0/topics/http/urls/
+Examples:
+Function views
+    1. Add an import:  from my_app import views
+    2. Add a URL to urlpatterns:  path('', views.home, name='home')
+Class-based views
+    1. Add an import:  from other_app.views import Home
+    2. Add a URL to urlpatterns:  path('', Home.as_view(), name='home')
+Including another URLconf
+    1. Import the include() function: from django.urls import include, path
+    2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
+"""
+from django.contrib import admin
+from django.urls import path, include
+
+urlpatterns = [
+    path("admin/", admin.site.urls),
+    path("", include("user.urls", namespace="user")),
```

Comment:
프로젝트 첫번째 path 인자는 app 이름으로 하는게 나아 보입니다.
```suggestion
    path("user", include("user.urls", namespace="user")),
```

Reply:
app 이름이 postings 인데 post로 하겠습니다.